products/rds-mysql.md

```diff
@@ -0,0 +1,66 @@
+---
+title: RDS mysql
+category: db
+iconSlug: mysql
+permalink: /rds-mysql
+releasePolicyLink: >-
+  https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MySQL.Concepts.VersionMgmt.html
+releaseImage: https://docs.aws.amazon.com/assets/r/images/aws_logo_dark.png
+activeSupportColumn: Active Support
```

Comment:
`activeSupportColumn` has been enabled but the `support` field is never defined on releases. That's what cause the error.

On https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MySQL.Concepts.VersionMgmt.html we can see there is only one support phase, so the `activeSupportColumn` is not needed an can be disabled with `activeSupportColumn: false`.

Reply:
Perfect that worked. I just omitted that line since it defaults to false.